gulpfile.js

```diff
@@ -158,6 +158,7 @@
                 .src('public/index.html')
                 .pipe(replace(/src="\/lib\/requirejs\/require\.js"/g, 'src="/application.js"'))
                 .pipe(replace(/href="\/css\/main\.css"/g, 'href="/main.css"'))
+                .pipe(replace(/<a href="(https?:\/\/)(.*)" class="cross-site-link">/g, '<a href="$1' + environment + '-$2" class="cross-site-link">'))
```

Comment:
This looks brittle - when the environment changes to `production` the links will all be broken. Either leave a TODO here  or store the base url prefix with the environment (or whatever other solution you can think of).